docker/Dockerfile-maven

```diff
@@ -1,4 +1,4 @@
-FROM kabanero/ubi8-openjdk:0.9.0
+FROM kabanero/ubi8-openjdk:0.9.1
```

Comment:
FROM kabanero/ubi8-openjdk:0.10.1  to match release?  https://github.com/kabanero-io/collections/releases/tag/0.10.1  I don't think anyone will care that the tag doesn't include that change as source..but they'd at least align